### PR TITLE
Super Staff Bros Melee: add m00ns' sig move & phrases

### PR DESCRIPTION
**Pull request by SolarisFox**
Also a minor style fix that I saw in there

### CODE REVIEW

**Comment by TheImmortal** [comment]
you could add his phrases as well

**Comment by SolarisFox** [comment]
Oh I didn't even notice those were missing.

**Comment by TheImmortal** [comment]
yeah i'm bad/lazy i didn't add it >_>

File: mods/seasonal/moves.js [hunk 1]
```diff
@@ -147,6 +148,22 @@ exports.BattleMovedex = {
 		target: "normal",
 		type: "Dark",
 	},
+	// m00ns
+	oh: {
+		num: -568;
```
Comment (SolarisFox) [inline mods/seasonal/moves.js]: Wherein I break everything with my carelessness

**Comment by TheImmortal** [comment]
Hah

**Pull request merged**